x-pack/plugins/rollup/rollup_data_enricher.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const rollupDataEnricher = async (indicesList, callWithRequest) => {
+  if (!indicesList || !indicesList.length) {
+    return indicesList;
+  }
+  const params = {
+    path: '/_all/_rollup/data',
+    method: 'GET',
+  };
+  try {
+    const rollupJobData = await callWithRequest('transport.request', params);
```

Comment:
@bmcconaghy  I see that you use 'transport.request' here to  get the rollup_data 

But propably we should use the following factory for that. It's a 'legal' way to get the rollup data.
/x-pack/plugins/rollup/server/lib/call_with_request_factory/call_with_request_factory.js

FYI: @jen-huang, @timroes